modules/argocd-adding-sample-repository.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/using-argocd.adoc
+
+[id="argocd-adding-sample-repository_{context}"]
+= Adding the ArgoCD samples repository
```

Comment:
This module should probably be focused on generic instructions on how to add a Git repository to ArgoCD, not specifically the fork repo containing the samples.